packit/config/common_package_config.py

```diff
@@ -203,12 +209,23 @@ def __init__(
         # this is generated by us
         self.downstream_package_name: Optional[str] = downstream_package_name
         self._downstream_project_url: str = downstream_project_url
-        self.dist_git_base_url: str = dist_git_base_url or getenv(
-            "DISTGIT_URL", PROD_DISTGIT_URL
-        )
-        self.dist_git_namespace: str = dist_git_namespace or getenv(
-            "DISTGIT_NAMESPACE", DISTGIT_NAMESPACE
-        )
+        # TODO(Gris): Support mixing c9s and fedora-all branches
+        # TODO(Gris): Observed two PackageConfig been created with different
+        #             dist_git_branches, need investigation
+        if dist_git_branches and "c9s" in dist_git_branches:
+            self.dist_git_base_url: str = dist_git_base_url or getenv(
+                "DISTGIT_URL", CENTOS_STREAM_DISTGIT_URL
+            )
+            self.dist_git_namespace: str = dist_git_namespace or getenv(
+                "DISTGIT_NAMESPACE", CENTOS_STREAM_DISTGIT_NAMESPACE
+            )
+        else:
+            self.dist_git_base_url: str = dist_git_base_url or getenv(
+                "DISTGIT_URL", PROD_DISTGIT_URL
+            )
+            self.dist_git_namespace: str = dist_git_namespace or getenv(
+                "DISTGIT_NAMESPACE", DISTGIT_NAMESPACE
+            )
```

Comment:
I am wondering if this shouldn't be moved to the job implementation so we can have `None` as the default that can be replaced when we process a specific branch.

Reply:
I have no better option. Any suggestion?